app/mixins/connection-type.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+
+const {Mixin, inject} = Ember;
+
+export default Mixin.create({
+	fastboot: inject.service(),
+	connectionType: 'unresolved',
+
+	init() {
+		if (!this.get('fastboot.isFastBoot')) {
+			const connection = navigator.connection || navigator.mozConnection || navigator.webkitConnection;
+			const effectiveConnectionType = connection.effectiveType || connection.type;
```

Comment:
is effectiveType the same as type?
if not, I don't think we should log it as one value to not confuse us

Reply:
Is the same, but varies depending on API.

Reply:
@hakubo , maybe we should log both, since "effective" means "4g / 3g" and "type" means "wifi / cellular" ?

Reply:
yeah, logging both make sense to me